mergify_engine/context.py

```diff
@@ -1104,9 +1131,15 @@ class PullRequest:
         "check-success",
         "check-failure",
         "check-neutral",
+        "check-skipped",
+        "check-pending",
+        "check-stale",
         "status-success",
         "status-failure",
         "status-neutral",
+        "status-skipped",
+        "status-pending",
+        "status-stale",
```

Comment:
Can you remove the deprecated names.

Reply:
Which ones are deprecated?

Reply:
`status-` are deprecated, we use `check-` now.

Reply:
So remove all ``status-``?